### PR TITLE
perf(shim): Update kiennq-shim to v3.1

### PR DESCRIPTION
**Pull request by niheaven**
#### Description

Download from https://github.com/kiennq/scoop-better-shimexe/releases/latest

#### How Has This Been Tested?

VirusTotal:

https://www.virustotal.com/gui/file/554aac610eae2cd61cae3dea0c66fd8283e95eb9c4af5dd9e591197c6a28e119?nocache=1

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

File: supporting/shims/kiennq/version.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-2.2.1
```
Comment (chawyehsu) [inline supporting/shims/kiennq/version.txt:1]: Would be better to keep the file indicating what version is vendored.

Comment (niheaven) [reply supporting/shims/kiennq/version.txt:1]: Readd it

**Review by chawyehsu — CHANGES_REQUESTED** [review]
@kiennq Don't know if you have any plan of making a new release of the shim but we are going update the vendored one. Currently v3.0 is the planned but it was released in 2022. Are there any critical commits that need to be considered since then?

I'm blocking this PR to wait for response. Let's wait for 48 hrs at most.

**Comment by kiennq** [comment]
> Currently v3.0 is the planned but it was released in 2022. Are there any critical commits that need to be considered since then?

There's a commit that prevents the shim from requiring elevated launch unexpectedly when the shim name contains `update`, `install` string.
kiennq/scoop-better-shimexe@e0fb559a9a9e4c4c09b00d65d131519f4c9781d8

I've released a new version.

**Comment by niheaven** [comment]
Updated to v3.1

**Comment by chawyehsu** [comment]
Thanks for the quick response.

**Review by chawyehsu — APPROVED** [review]

**Pull request merged**